G1ANT.Addon.MSOffice/Commands/Excel/ExcelCloseCommand.cs

```diff
@@ -30,7 +30,7 @@ public void Execute(Arguments arguments)
         {
             try
             {
-                ExcelManager.RemoveInstance();
+                ExcelManager.CurrentExcel.Close();
```

Comment:
What is the state of `CurrentExcel` after this line? Isn't it kinda UB when you leave unusable instance in the `Manager`?